### PR TITLE
fix(menu, select): added MenuToggle to props table

### LINKED ISSUE #8211: [Dropdown next, Select next] - [add MenuToggle to props table]
**Is this a new component or an extension of an existing one?**
What is the existing component, if any?
MenuToggle is used in the examples.  Add it to the prop tables so consumers can easily find the documentation.

### PR DESCRIPTION
**Pull request by CodeDuchess**
**What**: Closes #8211

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/next/components/Select/examples/Select.md
Comment (jenny-s51) [inline packages/react-core/src/next/components/Select/examples/Select.md:5]:
```suggestion
propComponents: ['Select', SelectGroup, 'SelectOption', 'SelectList', 'MenuToggle']
```

File: packages/react-core/src/next/components/Dropdown/examples/Dropdown.md
Comment (jenny-s51) [inline packages/react-core/src/next/components/Dropdown/examples/Dropdown.md:5]:
```suggestion
propComponents: ['Dropdown', DropdownGroup, 'DropdownItem', 'DropdownList', 'MenuToggle']
```

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Thank you for your contribution @CodeDuchess, this is a great start!

- Can we update the PR title to match the syntax as recommended in our [Developers Guide](https://docs.google.com/document/d/1z6rDcWfovN9hLym4wcDiA_L8uCwNZzXvoJ8UztXDv1w/edit#)? E.g. "fix(menu, select): added MenuToggle to props table". This format gives us a good idea of what to expect in the PR. You can do this by clicking the "Edit" button to the right of the title.

- The issue number (#8211) can be moved to the description after "Closes". This will automatically close the corresponding issue once your PR is merged.

Left some code suggestions below as well, which will likely fix your build!

**Comment by nicolethoen** [comment]
This issue might be blocked until https://github.com/patternfly/patternfly-org/issues/3331 is addressed

File: packages/react-core/src/next/components/Dropdown/examples/Dropdown.md
Comment (CodeDuchess) [reply packages/react-core/src/next/components/Dropdown/examples/Dropdown.md]: @jenny-s51 Thanks so much, Jenny!

**Comment by CodeDuchess** [comment]
@nicolethoen @jenny-s51 I think all necessary edits have been made. I am accustomed to GitLab, so I am still getting used to this UI. So forgive me if I have overlooked something. Thanks so much for your assistance!!

**Review by jenny-s51 — COMMENTED** [review]
Awesome, this is really close now @CodeDuchess!

If you click the "Commit Suggestion" and then "Commit changes" for each of those two comments above, it should fix your build.

Then this will be good to go once https://github.com/patternfly/patternfly-org/issues/3331 is completed and merged.

**Comment by CodeDuchess** [comment]
@jenny-s51 This PR is ready for merge and publish. Thanks so much for your guidance and patience!

**Comment by nicolethoen** [comment]
Closing in favor of this issue being addressed in the v5 branch in the following PRs:
https://github.com/patternfly/patternfly-react/pull/8825
https://github.com/patternfly/patternfly-react/pull/8835

**Pull request closed without merging**
